src/node_http_parser_impl.h

```diff
@@ -152,11 +152,14 @@ struct StringPtr {
   size_t size_;
 };
 
+constexpr AsyncWrap::ProviderType DEFAULT_PROVIDER = \
+          AsyncWrap::PROVIDER_HTTPINCOMINGMESSAGE;
```

Comment:
I think this can be dropped, `new Parser` is not called without a fourth argument currently.